code/modules/surgery/mcomp_tendwounds.dm

```diff
@@ -0,0 +1,155 @@
+/datum/surgery/mcomp_wounds
+	name = "Tend Damage (Brute/Burn)"
+	possible_locs = ALL_LIMBS
+	invasiveness = list(SURGERY_DEPTH_SURFACE)
+	required_surgery_skill = SKILL_SURGERY_YAUTJA // Only predators can do this
+	pain_reduction_required = 0
+	steps = list(
+		/datum/surgery_step/mstabilize_wounds,
+		/datum/surgery_step/mtend_wounds,
+		/datum/surgery_step/cauterize/mclamp_wound,
+				)
+	lying_required = FALSE
+	self_operable = TRUE
+
+/datum/surgery/mcomp_wounds/can_start(mob/living/carbon/human/user, mob/living/carbon/human/patient, var/obj/limb/L, obj/item/tool)
+	if((istype(user) && HAS_TRAIT(user, TRAIT_YAUTJA_TECH)) && (patient.getBruteLoss() || patient.getFireLoss()))
+		return TRUE
+	return FALSE
+
+//For closing an open incision first.
+/datum/surgery/mcomp_wounds/open
+	invasiveness = list(SURGERY_DEPTH_SHALLOW)
+	steps = list(
+		/datum/surgery_step/mtend_wounds,
+		/datum/surgery_step/cauterize/mclamp_wound
+				)
+
+//------------------------------------
+
+/datum/surgery_step/mstabilize_wounds
+	name = "stabilize wounds"
+	desc = "stabilize the wounds"
+	tools = SURGERY_TOOLS_MEDICOMP_STABILIZE_WOUND
+	time = 5 SECONDS
+
+/datum/surgery_step/mstabilize_wounds/preop(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool, tool_type, datum/surgery/surgery)
+	user.affected_message(target,
+		SPAN_NOTICE("You begin to stabilize your wounds with \the [tool]."),
```

Comment:
since this can be done on other people too, it needs to refer to the target and not just "your"